modules/turn/turn.c

```diff
@@ -341,22 +341,41 @@ static void tmr_handler(void *arg)
 {
 	struct tmr *tmr = arg;
 	struct le *le;
+	int thrd_id;
 
 	mtx_lock(&turndp()->mutex);
 	if (!turndp()->run)
 		re_cancel();
 
+	thrd_id = GetThreadId(thrd_current());
```

Comment:
Better use `thrd_current()` and `thrd_equal()`, `GetThreadId` is windows only

Reply:
TKS. thrd_current() in the Windows API returns a pseudo-handle to the current thread, and multiple invocations will return the same pseudo-handle. GetThreadId can correctly obtain the value. I will try another method.